workflow-templates/im-test-cypress-schedule.yml

```diff
@@ -0,0 +1,203 @@
+#         old: "cy:ci" : "....&& cypress run --project ./ --reporter ./mocha-teamcity-reporter \"",
+#         new: "cy:ci" : "....&& cypress run --project ./ --reporter mochawesome --reporter-options overwrite=false,html=false,json=true \"",
+#      3. Add the following pre-ci script.  Your name will vary based on your cypress ci script name.  This removes results between test runs.
+#         "precy:ci": "rm -rf mochawesome-report/",
+#      4. Add the following script that merges all test json files and generates an html report.  Your name may vary based on your project conventions.  If the output file or results dir are changed, update the env values in the workflow.
+#         "cy:report": "mochawesome-merge mochawesome-report/*.json > merged-results.json && marge --reportDir cypress-results merged-results.json",
+#                                                                                             ^^ marge is not a typo, it's a command for the "mochawesome report generator"
+
+name: 🧪 Run Cypress on Schedule
+run-name: 🧪 Run Cypress on Schedule
+
+# TODO: Verify Triggers
+on:
+  schedule: # TODO: Update the schedule to run at the desired frequency. Do not run more than every 30 minutes as to not produce undue load on the system.
+    - cron: '59 * * * *' # Run Every hour # TODO: Update the frequency of the schedule.  Do not run more than every 30 minutes as to not produce undue load on the system.
+    # - cron: '1/30 * * * *' # Run Every 30 minutes # TODO: Uncomment this line if you want to run every 30 minutes.
+    # - cron: '1/40 * * * *' # Run Every 40 minutes # TODO: Uncomment this line if you want to run every 40 minutes.
+
+env:
+  GITHUB_REF: 'main' # Default branch to run against.
+  READ_PKG_TOKEN: ${{ secrets.READ_PKG_TOKEN }} # This is an org-level secret
+  CYPRESS_TEST_DIR: '' # TODO: Set to the directory containing the js tests like tests/cypress.
+  NODE_VERSION: '' # TODO: Update this to the version of node you want to use for the tests
+  CYPRESS_CI_SCRIPT: '' # TODO: Add cypress script name, example: 'cy:run:healthcheck:dev'.
+  CYPRESS_REPORT_SCRIPT: 'cy:report' #TODO:  Add the name of the npm script that merges the files and creates an html report, if you copied the one above it is cy:report
+  CYPRESS_RESULTS_FILE: 'merged-results.json' # TODO: Update this value if it was added as something different in the npm report script
+  CYPRESS_REPORT_DIR: 'cypress-results' # TODO: Update this value if it was added as something different in the npm report script
+  CYPRESS_SCREENSHOTS: 'cypress/screenshots' # TODO: Update this value if it was added as something different in the npm report script
+  CYPRESS_NPM_PACKAGE_PATH: ${{ github.workspace }} # TODO: Update this to the path of the package.json file in the cypress test root directory
+  NPM_CACHE_PATH: ${{ github.workspace }}/node_modules # TODO: Update this to the path of the node_modules directory in the cypress test root directory
+
+jobs:
+  npm-cache:
```

Comment:
👍 Setting up a cache for this, since it seems like it's going to run pretty frequently based on the suggested cron values, is a good idea.